<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="remoteVideolabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="localVideolabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startMeetingButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>启动会议</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureImageButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cameraDevicecomboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="switchCameraButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>切换摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAudioButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始音频采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopAudioButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止音频采集</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableAudioSavecheckBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存音频</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableAudioPlaycheckBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放音频</string>
    </property>
   </widget>
   <widget class="QComboBox" name="audioDevicecomboBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startRecordButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRecordButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止录像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startReceiveButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopMeetingButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止会议</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
